pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -196,6 +196,15 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(cr *api.PerconaServerMo
 		members = append(members, member)
 	}
 
+	if cnf.Members.FixTags(members) {
```

Comment:
may be we should run `RemoveOld` first, because some members can become unavailable

Reply:
I think not. Because for failover scenario, user manually deletes the unavailable members from the config and that block won't run. And also we only change tags for existing members by comparing `rs.config()` and member slice populated by operator.

Reply:
but why this block won't run if the user manually deleted the unavailable members?

Reply:
sorry, I was referring to `RemoveOld` not `FixTags`